core/src/main/scala/io/finch/EndpointResult.scala

```diff
@@ -60,69 +46,49 @@ sealed abstract class EndpointResult[+A] {
     case _ => None
   }
 
-  /**
-   * Awaits for an [[Output]] wrapped with [[Try]] (indicating if the [[com.twitter.util.Future]]
-   * is failed).
-   *
-   * @note This method is blocking. Never use it in production.
-   *
-   * @return `Some(output)` if this endpoint was matched on a given input, `None` otherwise.
-   */
-  final def awaitOutput(d: Duration = Duration.Top): Option[Try[Output[A]]] = this match {
-    case EndpointResult.Matched(_, _, out) => Some(Await.result(out.liftToTry.run, d))
+  def awaitOutput(d: Duration = Duration.Top)(implicit e: Effect[F]): Option[Try[Output[A]]] = this match {
```

Comment:
Ideally, I'd like to move those methods to the test scope and use them implicitly.

Reply:
I actually like we have them. It makes experimenting with REPL a lot easier. I'd, perhaps, argue we should keep them in core.